src/resources/js/components/AccountCreation.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <v-container>
+    <h2 class="title">Inscription</h2>
+    <v-form
+        ref="form"
+        v-model="valid"
+        lazy-validation
+    >
+
+    <v-text-field
+            v-model="input.fname"
+            :rules="fnameRules"
+            required
+            label="Prénom"
+      />
+
+    <v-text-field
+            v-model="input.lname"
+            :rules="lnameRules"
+            required
+            label="Nom"
+      />
+
+      <v-text-field
+            v-model="input.email"
+            :rules="emailRules"
+            required
+            label="Adresse email"
+      />
+
+      <v-text-field
+            v-model="input.password"
+            :append-icon="showPassword ? 'midi-eye' : 'mdi-eye-off'"
+            :type="showPassword ? 'test' : 'password'"
+            :rules="passwordRules"
+            @click:append="showPassword = !showPassword"
+            required
+            label="Mot de passe"
+      />
+
+
+      <v-text-field
+            v-model="input.passwordConfirm"
+            :type="showPassword ? 'test' : 'password'"
+            required
+            label="Mot de passe"
+      />
+
+      <v-btn
+            :disabled="!valid"
+            color="success"
+            class="mr-4"
+            @click="register"
+            >
+            Créer compte
+      </v-btn>
+      <v-alert
+            v-model="message.show"
+            :class="message.type"
+            dismissible
+            >
+            {{ message.content }}
+      </v-alert>
```

Comment:
It'd be more appropriate if it was on top of the form.

Reply:
Changed in aaab904